documentation/book/assembly-kafka-connect.adoc

```diff
@@ -9,15 +9,18 @@
 [id='kafka-connect-{context}']
 = Kafka Connect
 
-The Cluster Operator deploys a link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] cluster, which can be used with your Kafka broker deployment.
-It is implemented as a `Deployment` with a configurable number of workers.
-The default image currently contains only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-The REST interface for managing the Kafka Connect cluster is exposed internally within the {ProductPlatformName} cluster as a `kafka-connect` service on port `8083`.
-
-Example `KafkaConnect` resources and the details about the `KafkaConnect` format for deploying Kafka Connect can be found in
-xref:assembly-deployment-configuration-kafka-connect-str[Kafka Connect cluster configuration]
-and
-xref:assembly-deployment-configuration-kafka-connect-s2i-str[Kafka Connect cluster with Source2Image support].
+link:https://kafka.apache.org/documentation/#connect[Kafka Connect^], a core component of Apache Kafka, is a tool for streaming high volumes of data into and out of your Kafka cluster. It provides a framework for building connectors (types of plug-ins) for your Kafka cluster, a REST API for managing connectors, and more. It includes two built-in connectors:
+
+|===
+| *Connector* | *Description*
+| `FileStreamSourceConnector` | Transfers data to your Kafka cluster from an external file (the source).
+| `FileStreamSinkConnector` | Transfers data from your Kafka cluster to an external file (the sink).
+|===
+
+In {ProductName}, you can use the Cluster Operator to deploy a Kafka Connect cluster to your {ProductPlatformName} cluster. Within your {ProductPlatformName} cluster:
+
+* A Kafka Connect cluster is implemented as a `Deployment` with a configurable number of workers.
```

Comment:
This list isn't valid for S2I enabled Connect deployment. I think you need to mention it here as well.